frontend/ui/widget/touchmenu.lua

```diff
@@ -823,6 +825,18 @@ function TouchMenu:onLastPage()
     return true
 end
 
+function TouchMenu:onPage(nb)
```

Comment:
Bad name for an event: is it to request going to a page ? is it emitted when gone to a page?